doc/conf.py

```diff
@@ -27,6 +27,7 @@
 html_sidebars = {'index': ['indexsidebar.html', 'searchbox.html']}
 html_additional_pages = {'index': 'index.html'}
 html_use_opensearch = 'http://sphinx-doc.org'
```

Comment:
I have no clue about how OpenSearch works, but this is probably the wrong URL?

This leads to a https://www.sphinx-doc.org/en/master/_static/opensearch.xml with:

```
<Url type="text/html" method="get"
       template="http://sphinx-doc.org/search.html?q={searchTerms}"/>
```

This URL doesn't lead to the search page.

This should probably be changed as well?

Reply:
Maybe yes. Please use `https://www.sphinx-doc.org/en/master` instead.

Reply:
I would prefer not to make this change in this PR, because it's a separate problem.

Also, as I said, I have no clue about OpenSearch, so I'd rather not touch this.

Reply:
Okay, I'll do that after merging.